server/templates/staff/course/assignment/assignment.html

```diff
@@ -191,13 +191,15 @@ <h3 class="box-title">Actions</h3>
                   <li> <a href="{{ url_for('.templates', cid=current_course.id, aid=assignment.id) }}" type="button">
                      <i class="fa fa fa-file-text-o"></i> Edit Assignment Template
                   </a></li>
-
                   <li> <a href="{{ url_for('.assign_grading', cid=current_course.id, aid=assignment.id) }}" type="button">
                         <i class="fa fa-user-plus"></i> Assign Grading
                   </a></li>
                   <li> <a href="{{ url_for('.assignment_queues', cid=current_course.id, aid=assignment.id) }}" type="button">
                         <i class="fa fa-inbox"></i> Grading Queues
                   </a></li>
+                  <li> <a href="{{ url_for('.publish_grades', cid=current_course.id, aid=assignment.id) }}" type="button">
+                        <i class="fa fa-wpforms"></i> Publish Grades
```

Comment:
I think we use the term "Scores" instead of Grades - so we should be consistent. (I might be wrong though)